index.html

```diff
@@ -5450,7 +5450,21 @@ <h2>Acknowledgments</h2>
     : "undecided";
   if (conclusionNode) processTag(conclusionNode, conclusionDict);
 
-  // TODO: extract the “supported use cases” & generate cross references
+  // extract the “supported use cases” & generate cross references
+  const capabilityUseCasesList = node.querySelector("ul[data-ucr-role='capability-uses']");
+  if (capabilityUseCasesList) {
+    const supportedUseCaseLinks = [...document.querySelectorAll(`[data-ucr-role='use-case'] [href='#${id}']`)];
```

Comment:
This may get some false positives from links in the description. There's a dedicated role on the required capability lists:
```suggestion
    const supportedUseCaseLinks = [...document.querySelectorAll(`[data-ucr-role='required-capabilities'] [href='#${id}']`)];
```